Mage/src/main/java/mage/abilities/dynamicvalue/AdditiveDynamicValue.java

```diff
@@ -42,8 +44,24 @@ public AdditiveDynamicValue copy() {
         return new AdditiveDynamicValue(this.dynamicValues);
     }
 
-    @Override
     public String getMessage() {
         return this.dynamicValues.stream().map(DynamicValue::getMessage).collect(Collectors.joining(" "));
     }
+
+    @Override
+    public String getMessage(ValuePhrasing phrasing) {
+        switch(phrasing){
+            case X_HIDDEN:
+                return "";
+            case FOR_EACH:
+                throw new IllegalArgumentException("FOR_EACH phrasing generation is not supported in AdditiveDynamicValue");
+            default:
+                return this.dynamicValues.stream().map(dv -> dv.getMessage(phrasing)).collect(Collectors.joining(" plus "));
+        }
+    }
+
+    @Override
+    public ValueHint getValueHint() {
+        return new ValueHint("total", this);
```

Comment:
I don't think "total" is ideal description here, is there a reason it can't use the generated message? there's a CardUtil method to capitalize if needed